packages/jest-config/src/__tests__/readInitialOptions.test.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import type {Config} from '@jest/types';
+import {readInitialOptions} from '../index';
+
+describe(readInitialOptions.name, () => {
```

Comment:
If I remember it right, this should work:

```suggestion
describe(readInitialOptions, () => {
```

Reply:
Ah this is pretty cool! Seems to work indeed.